mrp_subcontracting_skip_no_negative/models/stock_move.py

```diff
@@ -16,9 +16,9 @@ def _action_done(self, cancel_backorder=False):
         for move in moves_with_no_check:
             production_move = self.search([("move_dest_ids", "=", move.id)])
             production = production_move.production_id
-            if production.reservation_state != "assigned":
+            if production and production[0].reservation_state != "assigned":
```

Comment:
Wouldn't it be more appropriate to do something like this?
```suggestion
            if any(state != "assigned" for state in productions.mapped("reservation_state")):
```

Reply:
@yostashiro Could you check now? What I have done is to look for pending production moves. If there are several productions, I first try to assign the ones unassigned, and then check that they are all assigned